src/esp/sensor/VisualSensor.h

```diff
@@ -81,14 +90,51 @@ class VisualSensor : public Sensor {
     return false;
   }
 
+  /**
+   * @brief Sets resolution of Sensor's sensorSpec
+   */
+  void setResolution(int height, int width);
+  void setResolution(vec2i resolution);
+
   /**
    * @brief Returns RenderCamera
    */
   virtual gfx::RenderCamera* getRenderCamera() = 0;
 
+  /**
+   * @brief Gets near plane distance.
+   */
+  float getNear() { return near_; }
+
+  /**
+   * @brief Gets far plane distance.
+   */
+  float getFar() { return far_; }
+
+  /**
+   * @brief Returns the FOV of this Sensor
+   */
+  Mn::Deg getFOV() const { return hfov; }
+
  protected:
-  std::unique_ptr<gfx::RenderTarget> tgt_;
+  /** @brief projection parameters
+   */
+
+  /** @brief near clipping plane
+   */
+  float near_ = 0.001f;
 
+  /** @brief far clipping plane
+   */
+  float far_ = 1000.0f;
+
+  /** @brief field of view
+   */
+  Mn::Deg hfov = Mn::Deg{90.f};
```

Comment:
hfov_

Reply:
Also you can use literals here: 
You need: `using Mn::Math::Literals::operator""_degf;`
And then here you can use: `hfov_ = 90.0_degf;`

See an example here: https://github.com/facebookresearch/habitat-sim/blob/51b51e94c7fe645514758f28ada04cd49528997a/src/esp/gfx/test/DepthUnprojectionTest.cpp